cloudformation_templates/aws_monitoring.json

```diff
@@ -20,6 +20,28 @@
         "LogGroupName": {"Ref": "LogGroupName"},
         "RetentionInDays": {"Ref": "RetentionInDays"}
       }
+    },
+
+    "RDSSNSTopic": {
+      "Type": "AWS::SNS::Topic",
+      "Properties": {
+        "DisplayName": "RDSEvents",
+        "Subscription": [
+          {
+            "Protocol": "email",
+            "Endpoint": "support@digitalmarketplace.service.gov.uk"
```

Comment:
Let's move this to a variable in case we'll have different mailing lists for environments in the future